pkg/devfile/validate/events.go

```diff
@@ -0,0 +1,16 @@
+package validate
+
+import "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+
+func validatePreStart(preStart []string) (err error) {
+	return nil
+}
+
+func validateEvents(events v1alpha2.Events) (err error) {
+
+	if err := validatePreStart(events.PreStart); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
I don't understand the use of these functions? Or do you plan on using them later on?

Reply:
I forgot to remove them, they are unused now